app/controllers/reservations_controller.rb

```diff
@@ -10,17 +10,24 @@ def show
   end
 
   def create
-    reservation = Reservations::UseCases::Create.new.call(params: reservation_params)
+    ticket_desk = TicketDesks::Repository.new.find_by(params[:ticket_desk_id])
+    
+    if ticket_desk.type == "online"
+        reservation = Reservations::UseCases::CreateOnline.new.call(params: reservation_params.merge(status: "confirmed"))
+    else
+        reservation = Reservations::UseCases::CreateOffline.new.call(params: reservation_params.merge(client_id: 1))
```

Comment:
Why you always want to assign client with `id: 1`?

Reply:
I am thinking about having mock client for offline ticket desk.  So that there is always one user in database who is assigned to when someone buys ticket at the cinema and there is no user.

Reply:
You can have something like an unknown client record in DB for tickets sold offline. Or just don't associate client records in case of offline tickets.